prettyprinter/test/Testsuite/Main.hs

```diff
@@ -302,19 +302,21 @@ doNotRemoveLeadingWhitespaceText :: Assertion
 doNotRemoveLeadingWhitespaceText
   = let sdoc :: SimpleDocStream ()
         sdoc = SLine 0 (SText 2 "  " (SChar 'x' SEmpty))
-    in assertEqual "" sdoc (removeTrailingWhitespace sdoc)
+        sdoc' = SLine 2 (SChar 'x' SEmpty)
+    in assertEqual "" sdoc' (removeTrailingWhitespace sdoc)
 
 doNotRemoveLeadingWhitespaceChar :: Assertion
 doNotRemoveLeadingWhitespaceChar
   = let sdoc :: SimpleDocStream ()
         sdoc = SLine 0 (SChar ' ' (SChar 'x' SEmpty))
-    in assertEqual "" sdoc (removeTrailingWhitespace sdoc)
+        sdoc' = SLine 1 (SChar 'x' SEmpty)
+    in assertEqual "" sdoc' (removeTrailingWhitespace sdoc)
 
 doNotRemoveLeadingWhitespaceTextChar :: Assertion
 doNotRemoveLeadingWhitespaceTextChar
   = let sdoc :: SimpleDocStream ()
         sdoc = SLine 0 (SChar ' ' (SText 2 "  " (SChar 'x' SEmpty)))
-        sdoc' = SLine 0 (SText 3 "   " (SChar 'x' SEmpty))
+        sdoc' = SLine 3 (SChar 'x' SEmpty)
```

Comment:
Would changes like this one cause problems for anyone?! They seem nice from an efficiency perspective, but they aren't particularly important…

Reply:
Since this will be unnoticeable after rendering, I won't make a major version bump for this on its own.